core/domain/services/admin/registrations/list_table/csv_reports/CheckinsCSV.php

```diff
@@ -56,21 +54,20 @@ public static function getDatetineLabel(EE_Datetime $datetime)
      */
     public static function getCheckinValue(?EE_Checkin $checkin)
     {
-        $value = null;
         if ($checkin instanceof EE_Checkin && $checkin->get('CHK_in') === true) {
             /* translators: 1: check-in timestamp */
-            $value = sprintf(
+            return sprintf(
                 esc_html__('IN: %1$s', 'event_espresso'),
                 $checkin->get_datetime('CHK_timestamp', 'Y-m-d', 'g:i a')
             );
         } elseif ($checkin instanceof EE_Checkin && $checkin->get('CHK_in') === false) {
             /* translators: 1: check-in timestamp */
-            $value = sprintf(
+            return sprintf(
                 esc_html__('OUT: %1$s', 'event_espresso'),
                 $checkin->get_datetime('CHK_timestamp', 'Y-m-d', 'g:i a')
             );
         }
 
-        return $value;
+        return null;
```

Comment:
plz return an empty string in order to maintain data type consistency

Reply:
Fixed.